pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -224,13 +215,65 @@ func (r *ReconcilePerconaServerMongoDB) createSSLManualy(ctx context.Context, cr
 		Data: data,
 		Type: corev1.SecretTypeTLS,
 	}
-	err = r.client.Create(ctx, &secretObjInternal)
-	if err != nil && !k8serrors.IsAlreadyExists(err) {
-		return fmt.Errorf("create TLS internal secret: %v", err)
+	secretInternalUpdated, err := r.createOrUpdateSSLSecret(ctx, &secretObjInternal, certificateDNSNames)
+	if err != nil {
+		return errors.Wrap(err, "create TLS internal secret")
+	}
+	if secretUpdated || secretInternalUpdated {
+		return errors.New("ssl secrets updated")
 	}
 	return nil
 }
 
+func (r *ReconcilePerconaServerMongoDB) createOrUpdateSSLSecret(ctx context.Context, secret *corev1.Secret, DNSNames []string) (updated bool, err error) {
```

Comment:
either use these variables (`updated` is not used anywhere in the function) or remove the names

Reply:
fixed in b519ff017cfab23db834e04603c441a3dd41260a